### ISSUE TITLE
Key combinations with the Alt key (such as Alt+Tab) inappropriately show the keyboard shortcuts overlay

### ISSUE THREAD

**Issue by parcelcat** [ONLYOFFICE/DocumentServer#2907]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Linux (DEB package), Linux (RPM package)

### Version information

8.1.1.27

### Expected Behavior

When I switch from another application to OnlyOffice using Alt+Tab, OnlyOffice should appear on the screen without showing the [keyboard shortcuts overlay](https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/KeyboardShortcuts.aspx).

### Actual Behavior

OnlyOffice does sometimes show the [keyboard shortcuts overlay](https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/KeyboardShortcuts.aspx) when Alt+Tab is used to switch from another application to OnlyOffice.

### Reproduction Steps

1. Open a document in OnlyOffice.
2. Open another application.
3. Use Alt+Tab to switch from the other application to OnlyOffice.
4. OnlyOffice should appear without showing the [keyboard shortcuts overlay](https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/KeyboardShortcuts.aspx). However, OnlyOffice sometimes shows the overlay.

### Additional information

When I switch from another application to OnlyOffice using Alt+Tab, OnlyOffice often displays the [keyboard shortcuts overlay](https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/KeyboardShortcuts.aspx). This forces me to press the Alt key again to disable the overlay before I can start typing in the open document.

OnlyOffice should only show the keyboard shortcuts overlay when the user presses Alt by itself, not when the user presses Alt as a combination of multiple keys.

This issue is similar to #1660, except #1660 proposes a workaround to the problem (by allowing the user to disable the keyboard shortcuts overlay) instead of fixing the root problem (which is OnlyOffice inappropriately showing the keyboard shortcut overlay when a key combination is used). [This comment in #1660](https://github.com/ONLYOFFICE/DocumentServer/issues/1660#issuecomment-2102844141) shows that the issue also affects macOS.

**Comment by uc-ai2** [comment]
Hello, @parcelcat!
I was unable to reproduce the issue on Windows 10 (exe), POP_OS 22.04 (deb), as well as on virtual machines with Fedora 40 (rpm) and Mint 21.3 (deb). 
It’s possible that specific conditions are needed to reproduce the issue. What is your PC configuration and operating system?

**Comment by kayzzen01** [comment]
Hello @uc-ai2,

I use Arch + KDE and the problem occurs.

In OnlyOffice 8.1, I disabled the shortcut using the Alt key and it worked well for my workflow. Now in version 8.2 this option does not work, now the cursor is moved to the Home menu and I cannot type, needing to press the Alt key again. If I press the right arrow I will navigate in the OnlyOffice menu.

**Comment by EverardNisse** [comment]
Same behavior on Arch + KDE + Wayland

**Comment by maninmiddle** [comment]
Same behavior

**Comment by uc-ai2** [comment]
Hello, @parcelcat!

Thank you for noticing, the problem really exists. Issue No. 58345 in our issues tracker.

**Comment by Chasing1020** [comment]
I temporarily solved the problem by disabling "The Use Alt key to navigate the user interface using the keyboard option is used to enable using the Alt / Option key in keyboard shortcuts." in the advanced option. Alt shortcuts aren't necessary, at least not for me.
Ref: https://helpcenter.onlyoffice.com/onlyoffice-editors/onlyoffice-presentation-editor/helpfulhints/advancedsettings.aspx

**Comment by tukykarmakar** [comment]
I know it's been closed for a while, but I feel the need to mention that this wouldn't have been an issue if the shortcut overlay appeared when Alt is held instead of pressed once. Sometimes it's easier to use the shortcuts but you might not remember it, so holding down Alt to see the overlay is how it's usually done in other standardised applications.

**Comment by marco-calautti** [comment]
The issue is still present and makes switching between OnlyOffice and other apps a pain.

**Comment by tm-craggs** [comment]
Fixed this issue with the [PR](https://github.com/ONLYOFFICE/web-apps/pull/3183) above. It's my first one so would appreciate any feedback.

**Comment by vajrashura** [comment]
Yes, same issue with me. CachyOS and OnlyOffice 9.2.1.43 (x64 deb).

**Comment by z3us22** [comment]
I am still experiencing this issue on CachyOS.

**Comment by pdf** [comment]
> I know it's been closed for a while, but I feel the need to mention that this wouldn't have been an issue if the shortcut overlay appeared when Alt is held instead of pressed once. Sometimes it's easier to use the shortcuts but you might not remember it, so holding down Alt to see the overlay is how it's usually done in other standardised applications.

💯 this. I do not understand why this is implemented as a persistent toggle in onlyoffice instead of a momentary display, as in every other application.
